FastSimulation/Calorimetry/interface/CaloResponse.h

```diff
@@ -0,0 +1,50 @@
+// -*- C++ -*-
+//
+// Package:    FastSimulation/Calorimetry
+// Class:      CaloResponse
+//
+/**\class CaloResponser
+ Description: Returns scale from an TH3F histogram in a root file
+*/
+//
+// Original Author:  Maximilian Knut Kiesel
+//         Created:  Fri, 07 Aug 2015
+//
+//
+
+
+#ifndef CALORESPONSE_H
+#define CALORESPONSE_H
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "SimG4CMS/Calo/interface/CaloHitID.h"
+#include "FastSimulation/Particle/interface/RawParticle.h"
+
+#include <TH3F.h>
+#include <TFile.h>
+#include <TROOT.h>
+
+
+class CaloResponse {
```

Comment:
I think the class name still doesn't fully honour your method .
It's not limited to calorimetry,
and it fixes shape and scale.
Perhaps you can think of something better?
Not that this name is too important...